### PR TITLE
Message Serializable, and fix for MessageStoreSender

### PR DESCRIPTION
**Pull request by gvanbrakel**

### CODE REVIEW

File: core/src/main/java/nl/nn/adapterframework/pipes/MessageSendingPipe.java [hunk 1]
```diff
@@ -530,6 +530,7 @@ public PipeRunResult doPipe(Message input, IPipeLineSession session, IOutputStre
 				return wrapResult;
 			} else {
 				input = Message.asMessage(wrapResult.getResult());
+				// TODO must preserve input here, if result can be an InputStream
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/pipes/MessageSendingPipe.java]: Okey, so lets preserve it?

File: core/src/main/java/nl/nn/adapterframework/stream/Message.java [hunk 2]
```diff
@@ -364,4 +379,17 @@ public static String asString(Object object, String defaultCharset) throws IOExc
 		return Message.asMessage(object).asByteArray(defaultCharset);
 	}
 	
+	private void writeObject(ObjectOutputStream stream) throws IOException {
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/stream/Message.java]: Private method that's never been called?

File: core/src/main/java/nl/nn/adapterframework/stream/Message.java [hunk 3]
```diff
@@ -36,8 +39,11 @@
 import nl.nn.adapterframework.util.StreamUtil;
 import nl.nn.adapterframework.util.XmlUtils;
 
-public class Message {
-	protected Logger log = LogUtil.getLogger(this);
+public class Message implements Serializable {
+
+	private static final long serialVersionUID = 1L;
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/stream/Message.java]: What's the version?

File: core/src/main/java/nl/nn/adapterframework/stream/Message.java (hunk 2 above)
Comment (gvanbrakel) [reply core/src/main/java/nl/nn/adapterframework/stream/Message.java]: Zo werkt dat met Serializable

File: core/src/main/java/nl/nn/adapterframework/stream/Message.java (hunk 3 above)
Comment (gvanbrakel) [reply core/src/main/java/nl/nn/adapterframework/stream/Message.java]: ok, ok, ...

**Pull request merged**